docs/UserGuide.md

```diff
@@ -323,23 +338,26 @@ Examples:
 
 ### Undoing latest command : `undo`
 
-Undoes the changes from the latest command.
+Undoes the changes made from the last command if it was successfully executed, partially or fully.
 
 Format: `undo`
 
 * Commands that can be undone:
-    * `add`
-    * `delete`
-    * `newtag`
-    * `deletetag`
-    * `renametag`
-    * `tag`
-    * `untag`
-    * `edit`
-    * `filter`
-    * `find`
-    * `setrsvp`
-* Only the **latest** successfully executed command can be undone.
+  * `add`
+  * `delete`
+  * `newtag`
+  * `deletetag`
+  * `renametag`
+  * `tag`
+  * `untag`
+  * `edit`
+  * `filter`
+  * `find`
+  * `setrsvp`
+* Only the **latest** command can be undone, if it was successfully executed. 
+* Some examples of the `undo` command are:
```

Comment:
Maybe this can be changed to 'Examples:' to standardize with the rest of the commands